dynamiq/storages/vector/dry_run.py

```diff
@@ -0,0 +1,96 @@
+from pydantic import BaseModel, ConfigDict, Field
+
+from dynamiq.utils.logger import logger
+
+
+class DryRunConfig(BaseModel):
```

Comment:
Let's move this config to more proper place as theoretically it not just about Vector stores (we have types, base and so on). Please check for proper place
Let's also move `class DryRunMixin` also in more proper way as it used by base Node